Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/Card.h

```diff
@@ -59,6 +59,8 @@ namespace AzQtComponents
             qreal disabledIconAlpha;    //!< Alpha value for disabled icons. Must be a value between 0.0 and 1.0.
         };
 
+        static void applySeparatorStyle(Card* card);
```

Comment:
nit: For all O3DE code we prefer to use `UpperCamelCase` naming so I believe this should ideally be `ApplySeparatorStyle` to be consistent (we do use lowercase when overriding Qt member functions of course 😛)

Reply:
I think all other Components use `applySeparatorStyle`, so I'd be ok with keeping this for a separate pass.

Reply:
Ah okay cool please ignore this comment then :+1: